src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModule.cs

```diff
@@ -118,6 +119,11 @@ public ResultWrapper<GethLikeTxTrace> debug_traceTransactionInBlockByIndex(byte[
             return ResultWrapper<GethLikeTxTrace>.Success(transactionTrace);            
         }
 
+        public async Task<ResultWrapper<bool>> debug_setReceiptMigrationPoint(long blockNumber)
+        {
+            return ResultWrapper<bool>.Success(await _debugBridge.SetReceiptMigrationPoint(blockNumber + 1));
```

Comment:
this blockNumber, blockNumber + 1 may lead to confusion?

Reply:
adding comment (inclusive->exclusive conversion)